docs/eventing/sources/pingsource.md

```diff
@@ -1,196 +1,257 @@
   name: test-ping-source
 spec:
-  schedule: "*/1 * * * *"
+  schedule: "*/2 * * * *"
   contentType: "application/json"
   data: '{"message": "Hello world!"}'
   sink:
     ref:
-      apiVersion: v1
+      apiVersion: serving.knative.dev/v1
       kind: Service
       name: event-display
 EOF
 ```
-
 {{< /tab >}}
 
 {{% tab name="kn" %}}
-Notice that the namespace is specified in two places in the command in `--namespace` and the `--sink` hostname
+Create a PingSource by using the `kn` CLI:
+
 ```shell
 kn source ping create test-ping-source \
-  --namespace pingsource-example \
-  --schedule "*/1 * * * *" \
-  --data '{"message": "Hello world!"}' \
-  --sink http://event-display.pingsource-example.svc.cluster.local
+  --schedule "*/2 * * * *" --data \
+  '{ "message": "Hello world!" }' \
+  --sink ksvc:event-display
 ```
-
 {{< /tab >}}
-{{< /tabs >}}
 
-## (Optional) Create a PingSource with binary data
+{{% tab name="binary" %}}
+You can send binary data, that cannot be directly serialized by using YAML, to downstream, by using
+`dataBase64` as the payload. `dataBase64` carries data that is base64-encoded.
 
-Sometimes you may want to send binary data, which cannot be directly serialized in yaml, to downstream. This can be achieved by using `dataBase64` as the payload. As the name suggests, `dataBase64` should carry data that is base64 encoded.
+**Note:** `data` and `dataBase64` cannot co-exist.
 
-Please note that `data` and `dataBase64` cannot co-exist.
+Create the event source with binary data from standard in by running:
 
 ```shell
-kubectl create -n pingsource-example -f - <<EOF
+cat <<EOF | kubectl create -f -
 apiVersion: sources.knative.dev/v1beta2
 kind: PingSource
 metadata:
   name: test-ping-source-binary
 spec:
-  schedule: "*/1 * * * *"
+  schedule: "*/2 * * * *"
   contentType: "text/plain"
   dataBase64: "ZGF0YQ=="
   sink:
     ref:
-      apiVersion: v1
+      apiVersion: serving.knative.dev/v1
       kind: Service
       name: event-display
 EOF
 ```
+{{< /tab >}}
+{{< /tabs >}}
+
+
+## Verify the message was sent
+
+Verify that the message was sent to the Knative Eventing system by looking at message dumper logs.
 
-### Verify
+{{< tabs name="event-display" default="kubectl" >}}
+{{% tab name="kubectl" %}}
 
-View the logs for the `event-display` event consumer by
-entering the following command:
+View the logs of the event-display service by running:
 
 ```shell
-kubectl -n pingsource-example logs -l app=event-display --tail=100
+kubectl logs -l serving.knative.dev/service=event-display -c user-container --since=10m
 ```
 
-This returns the `Attributes` and `Data` of the events that the PingSource sent to the `event-display` Service:
+{{< /tab >}}
+{{% tab name="kail" %}}
+
+Use `kail` to tail the logs of the subscriber by running:
+
+```shell
+kail -l serving.knative.dev/service=event-display -c user-container --since=10m
+```
+
+For more information, see the [`kail`](https://github.com/boz/kail) repository in GitHub.
```

Comment:
I'd call this out earlier as an optional prereq, e.g. "if you want to view logs, you need to install kail" and add the link there, so it's not being introduced mid procedure

Reply:
+1

You could also reference using log aggregation via https://knative.dev/docs/install/collecting-logs/ as another way to view the logs if desired, but I'm not super-happy with that yet so I don't want to add it to this PR.